### ISSUE TITLE
Parse Syntax error with AngularJS 1.3.5

### ISSUE THREAD

**Issue by aug70** [esvit/ng-table#496]
I'm using 
* AngularJs 1.3.5
* http://bazalt-cms.com/assets/ng-table/0.3.0/ng-table.js

Getting below error from
*ng-table.js* 

```
Error: [$parse:syntax] Syntax Error: Token '.' is unexpected, expecting [:] at column 148 of the expression [{ 'sortable': column.sortable, 'sort-asc': params.sorting()[column.sortable]=='asc', 'sort-desc': params.sorting()[column.sortable]=='desc', column.class: true }] starting at [.class: true }].
```

Below code in "ng-table.js" is causing the problem
```
column.class: true
```

**Comment by iyel** [comment]
Is error present in the latest release?

**Comment by aug70** [comment]
Seems fixed in v0.3.1, thank you
